### PR TITLE
Corrected typographical error

### PR DESCRIPTION
**Pull request by dj107**
Changed experiance to experience on line 22.

### CODE REVIEW

**Review by Sarthak160 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ npm i -g yarn
 yarn install
 ```
 
-- Furthermore, to commit your changes use `yarn commit` instead of `git commit` for better commit experiance.
+- Furthermore, to commit your changes use `yarn commit` instead of `git commit` for better commit experience.
```
Comment (Akshit42-hue) [inline README.md:22]: Further more, to commit your changes use `yarn commit` instead of `git commit` for better commit experience.

Comment (dj107) [reply README.md:22]: ok

Comment (dj107) [reply README.md:22]: Furthermore is written together in this case, that is a complete word, that is correct

Comment (Akshit42-hue) [reply README.md:22]: Oh Yes

**Review by Akshit42-hue — APPROVED** [review]
LGTM, Great work @dj107

**Pull request merged**